test/cppsuite/configs/hs_cleanup_stress.txt

```diff
@@ -1,6 +1,6 @@
 # Run for half an hour.
 duration_seconds=1800,
-cache_size_mb=200,
+cache_size_mb=2000,
```

Comment:
Can you add a comment explaining the value of the cache here? It can be exactly what you said in the PR description.